learn/safe-core-account-abstraction-sdk/protocol-kit/README.md

```diff
@@ -1,7 +1,290 @@
+    2. Perform an off-chain signature of the transaction before proposing
+    3. Submit the transaction and signature to the Safe Transaction Service
+2. **Owner 2 confirms the transaction**
+    1. Get pending transactions from the Safe service
+    2. Perform an off-chain signature of the transaction
+    3. Submit the signature to the service
+3. **Anyone executes the transaction**
+    1. In this example, Owner 1 executes the transaction
+    2. Anyone can get the pending transaction from the Safe service
+    3. Account executing the transaction pays the gas fee
+
+### Create a Transaction
+
+For more details on what to include in a transaction see, [Create a Transaction in the Safe Core SDK Guide](https://github.com/safe-global/safe-core-sdk/blob/main/guides/integrating-the-safe-core-sdk.md#4-create-a-transaction).
+
+```tsx
+import { SafeTransactionDataPartial } from '@safe-global/safe-core-sdk-types'
+
+// Any address can be used. In this example you will use vitalik.eth
+const destination = '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045'
+const amount = ethers.utils.parseUnits('0.005', 'ether').toString()
+
+const safeTransactionData: SafeTransactionDataPartial = {
+  to: destination,
+  data: '0x',
+  value: amount
+}
+// Create a Safe transaction with the provided parameters
+const safeTransaction = await safeSdkOwner1.createTransaction({ safeTransactionData })
+```
+### Propose a Transaction
+
+To propose a transaction to the Safe Transaction Service we need to call the method `proposeTransaction` from the Safe Service Client instance.
+
+For a full list and description of the properties that `proposeTransaction` accepts, see [Propose the transaction to the service](https://github.com/safe-global/safe-core-sdk/blob/main/guides/integrating-the-safe-core-sdk.md#propose-transaction) in the Safe Core SDK guide.
+
+```tsx
+// Deterministic hash based on transaction parameters
+const safeTxHash = await safeSdkOwner1.getTransactionHash(safeTransaction)
+
+// Sign transaction to verify that the transaction is coming from owner 1
+const senderSignature = await safeSdkOwner1.signTransactionHash(safeTxHash)
+
+await safeService.proposeTransaction({
+  safeAddress,
+  safeTransactionData: safeTransaction.data,
+  safeTxHash,
+  senderAddress: await owner1Signer.getAddress(),
+  senderSignature: senderSignature.data,
+})
+```
+
+### Get Pending Transactions
+
+Recall that you created the `safeService` in [Initialize the Safe Service Client](#initialize-the-safe-service-client).
+
+```tsx
+const pendingTransactions = await safeService.getPendingTransactions(safeAddress).results
+```
+
+### Confirm the Transaction: Second Confirmation
+
+Owner 2 needs a different Safe object. However, you don’t need to create it with the Safe factory. You can create it with the `create` method of the Safe object. The Safe smart contract is already live on the blockchain so you can just pass the Safe address used when you created the Safe.
```

Comment:
```suggestion
When owner 2 is connected to the application, the Safe Core SDK should be initialized again with the existing Safe address the address of the owner 2 instead of the owner 1.
```